src/databricks/labs/remorph/config.py

```diff
@@ -14,7 +14,8 @@ class TranspileConfig:
     __file__ = "config.yml"
     __version__ = 1
 
-    source_dialect: str
+    transpiler: str
```

Comment:
Should we default to our sqlglot path?

Reply:
Done here: https://github.com/databrickslabs/remorph/pull/1315/files?short_path=b335630#:~:text=transpiler%20%3D%20self,directory/file)%22)